lib/std/os/linux.zig

```diff
@@ -1259,6 +1259,10 @@ pub fn fdatasync(fd: fd_t) usize {
     return syscall1(.fdatasync, @bitCast(usize, @as(isize, fd)));
 }
 
+pub fn prctl(option: i32, arg2: c_ulong, arg3: c_ulong, arg4: c_ulong, arg5: c_ulong) usize {
+    return syscall5(.prctl, @bitCast(usize, @as(isize, option)), arg2, arg3, arg4, arg5);
+}
```

Comment:
The `option` argument could probably be made unsigned as every valid value is positive. I think we need to use `c_ulong` for the other args for correctness, but I would love for someone to tell me we could use `usize` or something else instead.

Reply:
On Linux, c_ulong is always the same as `usize`, so that's why we can use usize here, since we are in linux.zig :)

https://github.com/ziglang/zig/blob/42c32dbc7b6cb474a1aa6dc524acb7451efeb15e/src-self-hosted/type.zig#L2994-L3011

Reply:
Wonderful!